docs/site/WEBSTORM.md

```diff
@@ -0,0 +1,85 @@
+# Developing with Webstorm
+
+There is currently an issue opened with Webstorm :
+[WEB-34416](https://youtrack.jetbrains.com/issue/WEB-34416)
+
+It should be fixed in `2018.3`, WebStorm is indexing symlinks create by Lerna
+inside `node_modules`, so it takes ages. As a workaround you need to exclude
+every `node_modules` directory or you can use [VisualStudio Code](./VSCODE.md).
+
+We recommend our contributors to use
+[WebStorm](https://www.jetbrains.com/webstorm/) with the following extensions
+installed:
+
+- [Prettier - Code Formatter](https://prettier.io/docs/en/webstorm.html) for
+  automatic formatting of source files on save.
+
+We also recommend our contributors to enable
+[TSLint](https://www.jetbrains.com/help/webstorm/2018.2/tslint.html) in WebStorm
+to highlight and auto-fix linting problems directly in the editor. Make sure you
+are using the TSLint package from loopback-next
+`loopback-next/packages/build/node_modules/tslint`.
+
+## How to verify TypeScript setup
+
+### Compilation errors
+
+1.  Open any existing TypeScript file, e.g. `packages/src/index.ts`
+
+2.  Add a small bit of code to break TypeScript's type checks, for example:
+
+    ```ts
+    const foo: number = 'bar';
+    ```
+
+3.  Verify that WebStorm editor has marked `foo` with a red underscore. Hover
+    over `foo` and check the problem message. It should mention `[ts]` source,
```

Comment:
The text says "it should mention `[ts]` source", but the text message shown as an example does not contain `[ts]` string. I am not familiar with WebStorm anymore, is the `[ts]` source shown somewhere else in the UI? Should we change the proposed docs to say something else, e.g. "It should start with `TS` prefix"?

Reply:
Nice catch ! I missed that :)